client/src/components/PodcastTile.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+const PodcastTile = ({ podcast }) => {
+  return (
+    <div className="callout primary" key={podcast.id}>
```

Comment:
a `key` is only required within a `map()` iteration, so if we're mapping over data to create these tiles then the returned component needs the `key`